freeipa.spec.in

```diff
@@ -591,7 +591,7 @@ Requires: python2-sssdconfig
 Requires: cyrus-sasl-gssapi%{?_isa}
 Requires: ntp
 Requires: krb5-workstation >= %{krb5_version}
-Requires: authconfig
+Requires: authselect
```

Comment:
The issue described in https://bugzilla.redhat.com/show_bug.cgi?id=1560046 has been fixed in authselect 0.3.1. The latest version creates missing directories. Please change the requirement to ``Requires: authselect >= 0.3.1``

Reply:
done